test/nodetool/conftest.py

```diff
@@ -15,6 +15,8 @@
 from typing import NamedTuple
 
 import rest_api_mock
+sys.path.insert(1, sys.path[0] + '/../..')
```

Comment:
See above

Reply:
Same here, it's not an easy fix, it will require some refactoring.